### PR TITLE
Hw12

### PR DESCRIPTION
**Pull request by wuyntmm**
hw12 done

### CODE REVIEW

File: hw12/hw12_1.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import json
+
+try:
+    with open('json') as file:
```
Comment (acman) [inline hw12/hw12_1.py]: if you expect json data in file - name it something like `phonebook.json` using json extention

File: hw12/hw12_1.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    while True:
+
+        users_input = input('What do you want to do? ')
+
+        match users_input.split()[0]:
+            case 'stats':
+                stats()
+            case 'add':
+                if len(users_input.split()) > 2:
```
Comment (acman) [inline hw12/hw12_1.py]: you use `users_input.split()` a lot of times, better to set it to variable

**Review by acman — CHANGES_REQUESTED** [review]
LGTM! Only small fixes should be provided.

**Comment by wuyntmm** [comment]
Fixed

**Review by acman — APPROVED** [review]
nice job!

**Pull request merged**